palantir-java-format/src/main/java/com/palantir/javaformat/doc/Break.java

```diff
@@ -25,25 +24,19 @@
 import com.palantir.javaformat.Output;
 import com.palantir.javaformat.doc.State.BreakState;
 import java.util.Optional;
+import org.immutables.value.Value;
 
 /** A leaf node in a {@link Doc} for an optional break. */
 @Immutable
-public final class Break extends Doc implements Op {
-    private final FillMode fillMode;
-    private final String flat;
-    private final Indent plusIndent;
-    private final Optional<BreakTag> optTag;
-
-    private Break(FillMode fillMode, String flat, Indent plusIndent, Optional<BreakTag> optTag) {
-        this.fillMode = fillMode;
-        this.flat = flat;
-        this.plusIndent = plusIndent;
-        this.optTag = optTag;
-    }
+@Value.Immutable
+public abstract class Break extends Doc implements Op {
```

Comment:
I converted this to an immutable thinking I wanted to add new fields to it but ended up not doing that.
Still, I prefer this style and it makes it easier to change in the future.